src/main/groovy/com/vanniktech/android/junit/jacoco/JunitJacocoExtension.groovy

```diff
@@ -28,4 +28,19 @@ class JunitJacocoExtension {
      * @since 0.6.0
      */
     boolean includeNoLocationClasses
+
+    /**
+     * Whether or not to use this plugin for configuring the Jacoco version and creating
+     * a task to combine the coverage reports for instrumentation and unit tests. The default
+     * value is true.
+     * @since 0.13.0
+     */
+    boolean configureInstrumentationCoverage = true
```

Comment:
This is enabled by default, but note that you need to manually set `testCoverageEnabled true` nonetheless

Reply:
Can we get it to set the `testCoverageEnabled` too?